src/reducers/channels.js

```diff
@@ -59,7 +62,7 @@ export default function reduce(state = initialState, action) {
         // we have to ensure that user isn't joined already.
         // https://github.com/ubergrape/chatgrape/issues/3804
         users: includes(users, userId) ? users : [...users, userId],
-        joined: isCurrentUser ? true : channel.joined
+        joined: isCurrentUser ? true : channel.joined,
```

Comment:
Can be simplified to `isCurrentUser || channel.joined`

Reply:
@glennreyes nice catch! thanks:)